w1thermsensor/core.py

```diff
@@ -167,6 +167,8 @@ def __init__(self, sensor_type=None, sensor_id=None):
                 )
             )
 
+        self.offset = 0.0
```

Comment:
Does it make sense allow providing an offset in the `__init__` ?

I don't like the pattern of :

```python
sensor = W1ThermSensor()
sensor.set_offset(...)
```

Reply:
I thought about this, and did it this way initially. In my application, I keep the `W1ThermSensor` objects around for a long time, and it makes sense to be able to calibrate them during the lifetime of the object.

Perhaps we can do both? An initial offset can be set in the constructor, and optionally set using `set_offset` ?

Reply:
Yes, sure - I was thinking about doing both :)

> An initial offset can be set in the constructor,

That should be optional, too - and `0.0` as a default

Reply:
Sounds good. I pushed a new version where I call `set_offset` from the constructor.